exercises/concept/role-playing-game/.docs/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Introduction
+
+## Null-References
+
+If you have ever used another programming language (C/C++, Python, Java, Ruby, Lisp, etc.), it is likely that you have encountered `null` or `nil` before.
```

Comment:
Python has `None` but not `null` or `nil` ... and `None` is not a null reference. It is a singleton. ;) I'm not sure if it fits the example well.